### PR TITLE
Server side rendering

### PR DESCRIPTION
**Pull request by seungprk**
Hi @rjgcabrera, can I ask you to review the changes to the module to allow for server side rendering by the proxy server? Main changes are that the top level module (entry point for webpack) had to be split out into two different files, one file targeting the server and one file targeting the client. Thank you 👍

### CODE REVIEW

**Comment by seungprk** [comment]
Hi @rjgcabrera, I changed the .render to .hydrate as you pointed out.

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "build:prod": "webpack -p",
     "start": "node ./server/server.js",
     "seed": "node ./db/seeding/generate.js data.json 10000000 && mongoimport --db apateez-reviews --collection stores --jsonArray ./db/seeding/data.json && rm ./db/seeding/data.json",
+    "seed:docker": "node ./db/seeding/generate.js data.json 10000000 && mongoimport --host database --db apateez-reviews --collection stores --jsonArray ./db/seeding/data.json && rm ./db/seeding/data.json",
```
Comment (rjgcabrera) [inline package.json:16]: Nice. The 'docker' npm script should be executing this, but you can add it in another feature branch for deployment.

**Review by rjgcabrera — APPROVED** [review]
Hey @seungprk 

The seed:docker script should be in the docker script, but everything else looks good. Cheers!

-Raymond

File: package.json (hunk 1 above)
Comment (seungprk) [reply package.json:16]: Thanks for pointing it out.

**Pull request merged**